content/300-guides/200-deployment/201-serverless/400-deploy-to-aws-lambda.mdx

```diff
@@ -27,7 +37,7 @@ binaryTargets = ["native", "rhel-openssl-1.0.x"]
 
 This is necessary because the runtime used in the development and when your application is deployed differ. Add the [`binaryTarget`](/reference/api-reference/prisma-schema-reference#binarytargets-options) to make the compatible Prisma engine file available.
```

Comment:
"in the deployment"

Reply:
```suggestion
This is necessary because the runtimes used during development and after deployment differ. Add the [`binaryTarget`](/reference/api-reference/prisma-schema-reference#binarytargets-options) to make the compatible Prisma engine file available.
```